portfolio/src/main/webapp/style.css

```diff
@@ -10,6 +10,25 @@ nav {
     border: solid;
     margin-bottom: 20px;
 }
+.content {
+  margin-left: auto;
+  margin-right: auto;
+  width: 650px;
+  text-align: center;
+}
+nav {
+    list-style-type: none;
+    background-color: lightblue;
+    border: solid;
+    margin-bottom: 20px;
```

Comment:
Consistent indenting again (2 spaces remains my recommendation; however, it should match what you use in all of your files, regardless of type).

Also line up your block definitions to be at the start of each line.